airavata-api/airavata-data-models/src/main/java/org/apache/airavata/model/status/ProcessState.java

```diff
@@ -42,7 +42,10 @@ public enum ProcessState implements org.apache.thrift.TEnum {
   COMPLETED(10),
   FAILED(11),
   CANCELLING(12),
-  CANCELED(13);
+  CANCELED(13),
```

Comment:
You must have updated status_models.thrift, but that file is missing from the PR.